### PR TITLE
Exercise 9

### PR DESCRIPTION
**Pull request by kallya21**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: .github/workflows/ci-pipeline.yml
Comment (JackMead) [inline .github/workflows/ci-pipeline.yml:25]: Don't forget to update this to main once you're merging 😄

File: .github/workflows/ci-pipeline.yml
Comment (JackMead) [inline .github/workflows/ci-pipeline.yml:35]: Nice work getting this in place in the end - for my records I'm assuming this is an unescaped URL?

Comment (JackMead) [reply .github/workflows/ci-pipeline.yml:35]: A nice bonus to add to the curl command here can be a `--fail` argument - without this curl reports success if the request gets a response, even if that response is a 4xx/5xx response!

**Review by JackMead — APPROVED** [review]
Nice work, I can see your pipeline is happily building & pushing your image, and the webhook is correctly notifying Azure now! 😄 Very happy for you to merge this & the M8 docs when you're ready

File: .github/workflows/ci-pipeline.yml
Comment (kallya21) [reply .github/workflows/ci-pipeline.yml:35]: Yes, it's an unescaped URL

**Pull request merged**
